classes/cocktail.js

```diff
@@ -4,11 +4,9 @@ class Cocktail {
     constructor(cocktail_obj) {
         this._cocktail = cocktail_obj;
     }
-
     get id() {
-        return this._cocktail.id;
+        return this._cocktail.id ;
```

Comment:
The semi-colon should be flush next to information, when in doubt use Shift+Alt+F to auto format code.